### PR TITLE
[#5987] improvement: Add more configurations for the config API

### LINKED ISSUE #5987: [Improvement] support 'gravitino.authorization.enable' value in '/configs' api
### What would you like to be improved?

If 'gravitino.authorization.enable' set 'fasle', cannot curl any access api successfully,
the information need add to the response of '/configs' api

### How should we improve?

_No response_

### PR DESCRIPTION
**Pull request by jerqi**
### What changes were proposed in this pull request?

Add more configurations for the config API

### Why are the changes needed?

Fix #5987 
Front end can use this config option to optimize the user experience.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

image

### CODE REVIEW

**Comment by jerqi** [comment]
cc @LauraXia123

**Review by mchades — DISMISSED** [review]

**Comment by tengqm** [comment]
Should the value be `"[Simple]"` or `["Simple"]`?

**Comment by jerqi** [comment]
> Should the value be `"[Simple]"` or `["Simple"]`?

You are right. It should be `["simple"]`.

**Comment by jerqi** [comment]
cc @lw-yang

File: server/src/main/java/org/apache/gravitino/server/web/ConfigServlet.java
Comment (mchades) [inline server/src/main/java/org/apache/gravitino/server/web/ConfigServlet.java:1]: suggest adding UT to cover the changes

Comment (jerqi) [reply server/src/main/java/org/apache/gravitino/server/web/ConfigServlet.java:1]: Added a UT.

**Review by mchades — APPROVED** [review]
LGTM.

**Comment by jerqi** [comment]
@tengqm Do you have other suggestion?

**Comment by tengqm** [comment]
> @tengqm Do you have other suggestion?

LGTM, although I was thinking of a JSON type configuration file for this purpose.

**Comment by jerqi** [comment]
> > @tengqm Do you have other suggestion?
> 
> LGTM, although I was thinking of a JSON type configuration file for this purpose.

We can expose more config options in the future. This config option is used for front end display. It's also useful to users if they use API to get the configurations.

**Pull request merged**